util/Migrator/MySql/Init.sql

```diff
@@ -0,0 +1,1028 @@
+    CONSTRAINT `FK_ProviderOrganizationProviderUser_ProviderOrganization_Provi~` FOREIGN KEY (`ProviderOrganizationId`) REFERENCES `ProviderOrganization` (`Id`) ON DELETE CASCADE,
+    CONSTRAINT `FK_ProviderOrganizationProviderUser_ProviderUser_ProviderUserId` FOREIGN KEY (`ProviderUserId`) REFERENCES `ProviderUser` (`Id`) ON DELETE CASCADE
+);
+
+CREATE INDEX `IX_Cipher_OrganizationId` ON `Cipher` (`OrganizationId`);
+
+CREATE INDEX `IX_Cipher_UserId` ON `Cipher` (`UserId`);
+
+CREATE INDEX `IX_Collection_OrganizationId` ON `Collection` (`OrganizationId`);
+
+CREATE INDEX `IX_CollectionCipher_CipherId` ON `CollectionCipher` (`CipherId`);
+
+CREATE INDEX `IX_CollectionGroups_GroupId` ON `CollectionGroups` (`GroupId`);
+
+CREATE INDEX `IX_CollectionUsers_OrganizationUserId` ON `CollectionUsers` (`OrganizationUserId`);
+
+CREATE INDEX `IX_CollectionUsers_UserId` ON `CollectionUsers` (`UserId`);
+
+CREATE INDEX `IX_Device_UserId` ON `Device` (`UserId`);
+
+CREATE INDEX `IX_EmergencyAccess_GranteeId` ON `EmergencyAccess` (`GranteeId`);
+
+CREATE INDEX `IX_EmergencyAccess_GrantorId` ON `EmergencyAccess` (`GrantorId`);
+
+CREATE INDEX `IX_Folder_UserId` ON `Folder` (`UserId`);
+
+CREATE INDEX `IX_Group_OrganizationId` ON `Group` (`OrganizationId`);
+
+CREATE INDEX `IX_GroupUser_OrganizationUserId` ON `GroupUser` (`OrganizationUserId`);
+
+CREATE INDEX `IX_GroupUser_UserId` ON `GroupUser` (`UserId`);
+
+CREATE INDEX `IX_OrganizationUser_OrganizationId` ON `OrganizationUser` (`OrganizationId`);
+
+CREATE INDEX `IX_OrganizationUser_UserId` ON `OrganizationUser` (`UserId`);
+
+CREATE INDEX `IX_Policy_OrganizationId` ON `Policy` (`OrganizationId`);
+
+CREATE INDEX `IX_ProviderOrganization_OrganizationId` ON `ProviderOrganization` (`OrganizationId`);
+
+CREATE INDEX `IX_ProviderOrganization_ProviderId` ON `ProviderOrganization` (`ProviderId`);
+
+CREATE INDEX `IX_ProviderOrganizationProviderUser_ProviderOrganizationId` ON `ProviderOrganizationProviderUser` (`ProviderOrganizationId`);
+
+CREATE INDEX `IX_ProviderOrganizationProviderUser_ProviderUserId` ON `ProviderOrganizationProviderUser` (`ProviderUserId`);
+
+CREATE INDEX `IX_ProviderUser_ProviderId` ON `ProviderUser` (`ProviderId`);
+
+CREATE INDEX `IX_ProviderUser_UserId` ON `ProviderUser` (`UserId`);
+
+CREATE INDEX `IX_Send_OrganizationId` ON `Send` (`OrganizationId`);
+
+CREATE INDEX `IX_Send_UserId` ON `Send` (`UserId`);
+
+CREATE INDEX `IX_SsoConfig_OrganizationId` ON `SsoConfig` (`OrganizationId`);
+
+CREATE INDEX `IX_SsoUser_OrganizationId` ON `SsoUser` (`OrganizationId`);
+
+CREATE INDEX `IX_SsoUser_UserId` ON `SsoUser` (`UserId`);
+
+CREATE INDEX `IX_Transaction_OrganizationId` ON `Transaction` (`OrganizationId`);
+
+CREATE INDEX `IX_Transaction_UserId` ON `Transaction` (`UserId`);
+
+CREATE INDEX `IX_U2f_UserId` ON `U2f` (`UserId`);
+
+INSERT INTO `__EFMigrationsHistory` (`MigrationId`, `ProductVersion`)
+VALUES ('20210617143411_InitPostgres', '5.0.5');
```

Comment:
@addisonbeck @djsmith85 `InitPostgres` in the MySql file, is it normal ? :)
Thx 👍

Reply:
@addisonbeck @djsmith85 ? :)
To be sure not to miss this...
Thank you again !